traffic_ops/app/conf/cdn.conf

```diff
@@ -31,7 +31,8 @@
         "backend_max_connections": {
             "mojolicious": 4
         },
-	"profiling_enabled": false
+        "whitelisted_oauth_urls": [],
```

Comment:
i can't decide if this should go inside of traffic_ops_golang or outside of it. maybe @dangogh has some ideas. not sure if the traffic_ops_golang node was reserved for certain properties or if they are arbitrary.

Reply:
I think if it's something that's directly related to the execution of `traffic_ops_golang`,  it should be inside.   This looks like something outside of that.   But,  scanning the list, it doesn't look like that was adhered to,  so that's not a strong opinion....